components/Card.js

```diff
@@ -1,9 +1,27 @@
 import styled from "styled-components";
+import { useState } from "react";
+import { Button } from "./Button";
 
 export default function Card({ voc }) {
+  const [showTranslation, setShowTranslation] = useState(false);
+
   return (
     <StyledSection>
-      <p>{voc.word}</p>
+      <p style={{ whiteSpace: "pre" }}>
```

Comment:
why "pre"? I think it is better to wrap a longer translation to fit the card.

Reply:
Good point! I need `white-space` to make the line break in the template string work in JSX. I have now changed it to `pre-wrap`, so the display of longer translations should also work, right?